### DISCUSSION TITLE
Actions for OnlyRead state

### DISCUSSION THREAD

**Discussion by DiegodeMiguel** [Telefonica/mistica-design#797] [Proposals]
Buenas!
Abro nuevo hilo para el caso de una RowList que tiene como acción un switch, me pide desarrollo como seria el caso para _OnlyRead_ ya que a ellos en el storybook les sale que debe ser toda la fila disable (es decir, text+action):
<img width="904" alt="image" src="https://user-images.githubusercontent.com/106736588/192481763-71f80f33-fa8a-411c-91ce-c0945a6bb452.png">

Desde mi punto de vista creo que tendría más sentido desactivar simplemente la acción ya que el usuario en el estado OnlyRead solo va poder ver la información, es decir si lo tiene activado, desactivado, si ha introducido alguna info addicional...etc 
Muestro el ejemplo sobre lo que debo aplicar ese OnlyRead:
<img width="905" alt="image" src="https://user-images.githubusercontent.com/106736588/192482623-098bf130-c8c7-445f-a32a-8a250c876832.png">
Thanks.

**Comment by aweell** [comment]
Buenas! @DiegodeMiguel,

Hasta la fecha se han propuesto dos vías para solucionar este problema:

* Utilizar una tag en el item de lista reemplazando al control con un texto que indique el estado
* Colocar un texto en el slot derecho del componente indicando el estado

Ejemplo: 
https://rebrand.ly/l73ofyo

**Reply by DiegodeMiguel** [reply]
Perfecto, muchas gracias! @aweell
